src/Configuration/Dotenv/ParsedVariable.hs

```diff
@@ -0,0 +1,55 @@
+module Configuration.Dotenv.ParsedVariable (ParsedVariable(..),
+                                            VName,
+                                            VValue(..),
+                                            VContents,
+                                            VFragment(..),
+                                            interpolateParsedVariables) where
+
+import Control.Monad (foldM)
+#if MIN_VERSION_optparse_applicative(0,13,0)
+import Data.Functor ((<$>))
+#endif
+import Control.Applicative ((<|>))
+import System.Environment (lookupEnv)
+
+data ParsedVariable
```

Comment:
I would still like to see a better naming scheme here. Let's at least rename the `V` prefix to `Var` or `Variable`.